napari/_qt/qt_main_window.py

```diff
@@ -1550,27 +1555,62 @@ def _screenshot(
         flash : bool
             Flag to indicate whether flash animation should be shown after
             the screenshot was captured.
-        size : tuple (int, int)
+        size : Optional[int, int]
             Size (resolution height x width) of the screenshot. By default, the currently displayed size.
             Only used if `canvas_only` is True.
-        scale : float
+        scale : Optional[float]
```

Comment:
```suggestion
        scale : float, optional
```
https://numpydoc.readthedocs.io/en/latest/format.html#parameters